src/helpers/routes.js

```diff
@@ -155,7 +155,8 @@ export function makeRoutes (baseRoutes, {
         path = adjustRouteDefinitionForTrailingSlash(path, trailingSlash, isChildWithRelativePath)
       }
 
-      if (shouldAddPrefix && isDefaultLocale && strategy === STRATEGIES.PREFIX && includeUprefixedFallback) {
+      //add a route without prefix as fallback
```

Comment:
Apart from introducing a linting issue, is this even needed. The variables are self-descriptive so this feel redundant.

Reply:
Not aware of your linting; I edited it in the browser but assume this is what you prefer?
```suggestion
      // add a route without prefix as fallback
```
Regarding obsoleteness of the comment, I just thought it's helpful to write down what will happen in the following lines (yes, plural) instead of expecting people to evaluate it in their head just to find out that this is not the closure they were looking for. There are many places in this file where a route gets pushed so I thought why not help with differentiation. :slightly_smiling_face:

Reply:
Either way should be fine
```suggestion
      // add a route without prefix as fallback
```
```suggestion
```
not sure if rearranging conditionals will do much since they still need to be seen in conjunction

Reply:
Since you are asking then just remove it.